src/pure.js

```diff
@@ -13,10 +15,11 @@ const render = (
   target = target || container.appendChild(document.createElement('div'))
 
   const ComponentConstructor = Component.default || Component
+  const isProps = !Object.keys(options).some(option => svleteComponentOptions.includes(option))
```

Comment:
Is this really safe?

Reply:
Should be.  Tests are passing.  What is your concern with it @alexkrolick ?

Reply:
Ye it's safe because Svelte doesn't update the client side options often. If they do then of course this would need to be updated. This was the easiest solution off the top of my head, I don't see any other way of doing it. Love to hear any thoughts :)

Reply:
Concern was that if your component has props that use any of the same names as those keys, it would go into "options" mode instead of "props" mode

Reply:
Ah yeah that's a good point.

Reply:
Ye I thought of that, but then I realised if that happens you just pass props in as an option (the old way) and you're good to go. I'd make a big assumption that the option keys being used as props are very unlikely.

Reply:
I guess that's something that should be mentioned in the docs. I'll take note and do it when I have free time.

Reply:
You could error if an unrecognized option is passed in "options" mode, that would catch when that mode is triggered accidentally, assuming you pass more props than just the overloaded key.

Reply:
Personally my opinion is it should be the way you normally use svelte.  That said I do like @alexkrolick  latest suggestion. 

I will say @mihar-22 in the future please try to give folks a chance to review before merging your own PR

Reply:
To add, no existing code will break because the only way people could pass in props before was via `{ props : { ... } }`, so if option names were used, then they'll be passed through just fine. When the developer refactors by passing props directly, they'll realise and revert that code back.

Reply:
That may be the case but anyone writing new code may experience a false positive or unexpected result in their test. It's a breaking change, actually. I'd suggest adding some error handling.

Reply:
I left the issue open for a very very long time for anyone to review and discuss. I just thought no one was interested. I heard back from some people that it'd be much easier to just pass the props in. The other options are very rarely used so it's a much better developer experience. No one wants to constantly write so much boilerplate which is exactly what Svelte aims to avoid itself.

Reply:
Really good suggestion @alexkrolick, I'll make a PR now for that.

Reply:
FYI for future reference issues tend to get less attention than PRs 😄

Reply:
Yeah the PR was only opened 2 hours ago

Reply:
NBD. As Alex said... "Going forward". 🙂